### ISSUE TITLE
Master password needs to be entered multiple times

### ISSUE THREAD

**Issue by stdevel** [stdevel/katprep#31]
Currently, the master password needs to be entered multiple times if an authentication container is used:
```
$ katprep_maintenance -C pinkepank.auth -s ewa.test.loc --insecure -n errata-snapshot*.json prepare
INFO:katprep_maintenance:This is just a SIMULATION - no changes will be made.
File password ('Foreman'@'ewa.test.loc'): 
File password ('Virtualization vc.test.loc'@'vc.test.loc'): 
...
```

Would be great to have caching.

**Comment by stdevel** [comment]
Might be related to issue #36, evaluating in branch `feature-password-param_issue36`.

**Comment by stdevel** [comment]
Fixed in commit 44a07192beb6ffa58a4a4ee3af9d5071d9810ca0.
